docs/tutorials/accelerate-pytorch/resnet-inferencing.md

```diff
@@ -0,0 +1,296 @@
+---
+nav_exclude: true
+---
+
+# Optimal Inferencing on Flexible Hardware with ONNX Runtime
+{: .no_toc }
+---
+
+As a developer who wants to deploy a PyTorch or ONNX model and maximize performance and hardware flexibility, you can leverage ONNX Runtime to optimally execute your model on your hardware platform.
+
+In this tutorial, you'll learn:
+
+1. how to use the [PyTorch](https://pytorch.org/vision/stable/models.html) ResNet-50 modelfor image classification
+2. convert to ONNX, and
+3. deploy to the default CPU, NVIDIA CUDA (GPU), and Intel OpenVINO with ONNX Runtime -- using the same application code to load and execute the inference across hardware platforms.
+
+[ONNX](https://onnx.ai/) was developed as the open-sourced ML model format by Microsoft, Meta, Amazon, and other tech companies to standardize and make it easy to deploy Machine Learning models on various types of hardware.[ONNX Runtime](https://onnxruntime.ai/) was contributed and is maintained by Microsoft to optimize ONNX model performance over frameworks like PyTorch, Tensorflow, and more. The ResNet-50 model is a commonly used model for image classification that is pretrained on ImageNet.
+
+This tutorial demonstrates how to run an ONNX model on CPU, GPU, and OpenVINO hardware with ONNX Runtime, using [Microsoft Azure Machine Learning](https://azure.microsoft.com/en-us/services/machine-learning/).
+
+## Setup
+
+- **OS Prerequisities:** Your environment should have [curl](https://curl.se/download.html) installed. This tutorial uses a Windows OS.
```

Comment:
I don't think there is anything specific to Windows in this tutorial. If that's the case we can remove the bit about Windows